Branch-SDK/src/io/branch/referral/BranchPreinstall.java

```diff
@@ -0,0 +1,105 @@
+package io.branch.referral;
+
+import android.content.Context;
+import io.branch.referral.Defines.PreinstallKey;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
```

Comment:
Compiler Warnings.

Reply:
@apeterson-branch what compiler warnings are you seeing?

Reply:
I think it was related to unused imports, I don't seem them anymore. In any case, these files are being used.